src/main/java/club/sk1er/patcher/hooks/ResourcePackRepositoryHook.java

```diff
@@ -11,15 +12,17 @@
 
 @SuppressWarnings("unused")
 public class ResourcePackRepositoryHook {
-    public static void updateRepositoryEntriesAll(ResourcePackRepository repository) {
+    public static void updateRepositoryEntriesAll(ResourcePackRepository r) {
```

Comment:
Rename this back to "repository" & make the variable accessor or something similar